config/core/test-configs.yaml

```diff
@@ -59,6 +59,12 @@ file_systems:
     type: debian
     ramdisk: 'buster-v4l2/20210503.0/{arch}/rootfs.cpio.gz'
 
+  debian_buster-libcamera_nfs:
+    type: debian-staging
+    ramdisk: 'buster-libcamera/20210603.0/{arch}/initrd.cpio.gz'
+    nfs: 'buster-libcamera/20210603.0/{arch}/full.rootfs.tar.xz'
+    root_type: nfs
```

Comment:
Please update to use this production image:
https://storage.kernelci.org/images/rootfs/debian/buster-libcamera/20210611.1/

(i.e. `type: debian` and the URL with the new date)